ambry-frontend/src/main/java/com.github.ambry.frontend/AmbrySecurityService.java

```diff
@@ -183,6 +185,7 @@ private void setHeadResponseHeaders(BlobInfo blobInfo, RestResponseChannel restR
     if (blobProperties.getContentType() != null) {
       restResponseChannel.setHeader(RestUtils.Headers.CONTENT_TYPE, blobProperties.getContentType());
     }
+    restResponseChannel.setHeader(RestUtils.Headers.ACCEPT_RANGES, "bytes");
```

Comment:
and HEAD usually indicates what GET will return. you might have to handle `RangeNotSatisfiable` in HEAD calls.